magento/magento2
# Prevent users being able to click multiple times on login button

### Description (*)

Prevent users being able to click multiple times on login button.

### Related Pull Requests


### Manual testing scenarios (*)

1. Go to the checkout
2. In the login form in checkout, try logging in.
3. While the loader is loading, click on the login button again.
4. After this patch, the login button should now be disabled while loading.

### Contribution checklist (*)
